### ISSUE TITLE
New LazyLoad feature does not play well with dynamic title. 

### ISSUE THREAD

**Issue by PolarbearDK** [softsimon/angular-2-dropdown-multiselect#298]
First of all, kudos to the new lazy load feature. Great stuff!

There is however a problem if you have the following options
```
{
    fixedTitle: false, 
    dynamicTitleMaxItems: 2, // Any value greater than 0 (0 is a workaround).
    isLazyLoad: true,
}
```
The problem is that only the selected items that is visible right now is displayed in the dynamic title.

Reproduced using [Lazy loading Plunker](https://plnkr.co/edit/fsZHbth4kzLI79hohcMG?p=preview):
- set `msSettings.fixedTitle = false`
- Run plunker and check first two countries
- Search for "cuba"

The dynamic title is now blank.

**Comment by softsimon** [comment]
@GregKass

**Comment by GregKass** [comment]
@softsimon @PolarbearDK: How important is the title ordering in this case? Case in point: Assume that the countries aren't necessarily alphabetically ordered, and are possibly arbitrarily ordered. You load the first twenty and check one. Then you search on countries containing "b", and scroll down a few loads and select another. Then you search on "c", scroll again, and select a third. How do you know where the first selection, second selection (b), and third selection (c) sit relative to one another? The first case is easy because it contains all selections, but where do "b" and "c" sit relative to one another if you've scrolled beyond the all known cases? You could possibly match cases that contain both "b" and "c" to map reference points, but this would only narrow things down a little. Would titles displaying in the order selected be acceptable? Maybe you could arrange them in the proper order when the selected titles have been loaded in without using a search parameter?

**Comment by PolarbearDK** [comment]
I don't think title ordering is important. It would be fine to display in the order they were selected.

**Comment by gund** [comment]
+1 for the selection order. Seems reasonable =)

**Comment by softsimon** [comment]
Should be fixed now in v1.6.1
